app/src/main/java/com/example/mytestproject/ui/weatherData/dailyWeather/TenDaysWeatherFragment.kt

```diff
@@ -60,10 +60,12 @@ class TenDaysWeatherFragment : Fragment() {
 
     }
 
-    private fun setTitle(cityName: String) {
-        (activity as? AppCompatActivity)?.supportActionBar?.title =
-            cityName.capitalize()
-        (activity as? AppCompatActivity)?.supportActionBar?.subtitle = getString(R.string.ten_day_text)
+    private fun setTitle() {
+        dailyWeatherViewModel.cityName.observe(viewLifecycleOwner, Observer{cityName ->
+            (activity as? AppCompatActivity)?.supportActionBar?.title =
```

Comment:
We can avoid duplicating code here.
Use (activity as? AppCompatActivity)?.supportActionBar?.run { ... }

Reply:
Fixed